maven/bnd-resolver-maven-plugin/pom.xml

```diff
@@ -84,7 +88,7 @@
                       <jsse.enableSNIExtension>false</jsse.enableSNIExtension>
                     </properties>
                     <goals>
-                        <goal>bnd-resolver:resolve</goal>
+                        <goal>package</goal>
```

Comment:
OK, but shouldn't the invoker invoke the `bnd-resolver:resolve` target? That is, should the test case look normal and the test runner run the test in the expected way?